qa/L0_simple_go_client/test.sh

```diff
@@ -50,12 +51,12 @@ RET=0
 
 # Generate Go stubs.
 rm -fr client common
-git clone https://github.com/triton-inference-server/client.git
+git clone ${TRITON_REPO_ORGANIZATION}/client.git
 go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
 
 pushd ${GO_CLIENT_DIR}
 git clone --single-branch --depth=1 -b $TRITON_COMMON_REPO_TAG \
-    https://github.com/triton-inference-server/common.git
+    ${TRITON_REPO_ORGANIZATION}/common.git
```

Comment:
Cannot suggest change because line is out of scope, but I believe line 64 should be:

`PACKAGE_PATH="${GOPATH}/src/${TRITON_REPO_ORGANIZATION}"`

Reply:
That change needed to be [reverted](https://gitlab-master.nvidia.com/dl/triton/server/-/commit/77176a8fb6751af92d295368672a460823ab8be6) due to how `go` expects the install path to be. In addition using the parameterized `${TRITON_REPO_ORGANIZATION}` would create a bad path as it would turn out something like `${GOPATH}/src/https://github.com/triton-inference-server`. See [this test](https://gitlab-master.nvidia.com/dl/dgx/tritonserver/-/jobs/72140414#L507) for an example of what I am talking about

Reply:
No fault on you here. I tunnel visioned the gitlab changes, should have realized what the resulting path would look like.

All good here then!